tests/integration/Cake.Common/ArgumentAliases.cake

```diff
@@ -0,0 +1,18 @@
+#load "./../scripts/xunit.cake"
+
+//////////////////////////////////////////////////////////////////////////////
+
+Task("Cake.Common.ArgumentAliases.HasArgument")
+    .Does(() =>
+{
+    // Given, When
+    var arg = Argument<string>("customarg");
+
+    // Then
+    Assert.Equal("hello", arg);
```

Comment:
No, this is passing. I pass the `--customarg=hello` to the Cake.exe call in the bootstrapper.